dvc/stage/exceptions.py

```diff
@@ -92,3 +92,39 @@ def __init__(self, missing_files):
 
         msg = "missing data '{}': {}".format(source, ", ".join(missing_files))
         super().__init__(msg)
+
+
+class StageNotFound(KeyError, DvcException):
+    def __init__(self, file, name):
+        super().__init__(
+            "Stage with '{}' name not found inside '{}' file".format(
+                name, file.relpath
+            )
+        )
+
+
+class StageNameUnspecified(DvcException):
+    def __init__(self, file):
+        super().__init__(
+            "Stage name not provided. "
+            "\nPlease specify the name as: `{0}:stage_name`".format(
```

Comment:
No need for a space before \n, but i don't think \n is needed neither.
```suggestion
            "Stage name not provided."
            "Please specify the name as: `{0}:stage_name`".format(
```